src/Attestation/Attestation.Test/ScenarioTests/AttestationController.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Microsoft.Azure.Commands.Common.Authentication;
```

Comment:
@shleiAmy please make sure that all `.cs` files in this PR have the appropriate license header

Reply:
all files have been added the license header